From 46e1cf4f174b9eb778f810272a9f9756b3e78d76 Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Tue, 8 Feb 2022 10:13:15 +0800
Subject: [PATCH] drivers: watchdog: restore watchdog default timeout value to
 30s

When run kexec/kdump feature, board resets suddenly, because
the watchdog timeout effects during try to start the second
kernel. This issue is introduced by commit 0ee65ee166bc("drivers:
watchdog: Enhance watchdog support in SPL for Stratix 10 and Agilex"),
it changes the watchdog timeour value to 10s. It is too short and
causes board resets when booting up the second kernel. So, restore
the default timeout value to 30s.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/watchdog/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/watchdog/Kconfig b/drivers/watchdog/Kconfig
index fc9bc8b7b8f0..569726119ca1 100644
--- a/drivers/watchdog/Kconfig
+++ b/drivers/watchdog/Kconfig
@@ -28,7 +28,7 @@ config WATCHDOG_TIMEOUT_MSECS
 	int "Watchdog timeout in msec"
 	default 128000 if ARCH_MX31 || ARCH_MX5 || ARCH_MX6
 	default 128000 if ARCH_MX7 || ARCH_VF610
-	default 10000 if ARCH_SOCFPGA
+	default 30000 if ARCH_SOCFPGA
 	default 16000 if ARCH_SUNXI
 	default 5376 if ULP_WATCHDOG
 	default 15000 if ARCH_BCM283X
-- 
2.44.1

